icu.yaml

```diff
@@ -124,18 +125,13 @@ subpackages:
         - merged-usrsbin
         - wolfi-baselayout
 
-# switch back to this once the GitHub release stops saying pre-release
-# github:
-#   identifier: unicode-org/icu
-#   strip-prefix: release-
 update:
   enabled: true
   manual: true # ICU updates contain ABI breaking changes which require manual intervention
```

Comment:
Do we want this package to continue to have `manual: true` in the update section?

Reply:
I think longer term no; but we need automation to perform the actual transitions when we get a new version.